sql/dbupdate.php

```diff
@@ -1555,14 +1555,41 @@
 ?>
 <#44>
 <?php
-// To apply new changes into WorkflowAP model as well as xoct_workflow table.
-//\srag\Plugins\Opencast\Model\Workflow\WorkflowAR::updateDB();
+// Re remove the column parameter from xoct_workflow if it's existent.
+if($ilDB->tableColumnExists('xoct_workflow', 'parameter')) {
```

Comment:
The field is called `parameters` and the fix has `parameter`
This also applies to the latest step!